### PR TITLE
feat: change contract verification config

### PR DESCRIPTION
**Pull request by JhChoy**
- Changed contract verification config to use the verification function of the 'hardhat-deploy' plugin.
- Ref: https://github.com/wighawag/hardhat-deploy#4-hardhat-etherscan-verify

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -121,6 +107,12 @@ const config: HardhatConfig = {
       saveDeployments: true,
       tags: ['mainnet', 'prod'],
       companionNetworks: {},
+      verify: {
+        etherscan: {
+          apiKey: 'API_KEY',
```
Comment (graykode) [inline hardhat.config.ts]: @JhChoy Can you change this key to `.env` file?

File: hardhat.config.ts [hunk 2]
```diff
@@ -12,7 +12,6 @@ import 'hardhat-gas-reporter'
 import 'hardhat-contract-sizer'
 import 'hardhat-abi-exporter'
 import 'solidity-coverage'
-import '@nomiclabs/hardhat-etherscan'
```
Comment (graykode) [inline hardhat.config.ts:15]: @JhChoy `import '@openzeppelin/hardhat-upgrades'` Why this one is missing?

Comment (JhChoy) [reply hardhat.config.ts:15]: Why is that needed?

File: hardhat.config.ts (hunk 1 above)
Comment (JhChoy) [reply hardhat.config.ts]: [d5b5dc5](https://github.com/clober-dex/mango-farmers-club/pull/7/commits/d5b5dc57c14a5d41e1ad918ea6ea2cfd5cd13058)

File: hardhat.config.ts (hunk 2 above)
Comment (graykode) [reply hardhat.config.ts:15]: https://docs.openzeppelin.com/upgrades-plugins/1.x/api-hardhat-upgrades#verify @JhChoy  It helps automatically verification in etherscan

Comment (JhChoy) [reply hardhat.config.ts:15]: What kinds of help? The contract verification command works now.
We are not using
```
npx hardhat verify --network mainnet PROXY_ADDRESS
```
We use
```
npx hardhat --network 1101 etherscan-verify
```
as I mentioned in the description of this PR.

**Review by graykode — APPROVED** [review]

**Pull request merged**